resolwe/elastic/management/commands/es_index.py

```diff
@@ -0,0 +1,27 @@
+""".. Ignore pydocstyle D400.
+
+=================
+Command: es_index
+=================
+
+"""
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from django.core.management.base import BaseCommand
+
+from resolwe.flow.models import Data, Sample
+
+from genesis.es_utils.utils import add_index
+
+
+class Command(BaseCommand):
+    """Build ElasticSearch indexes for data objects."""
+
+    help = "Build ElasticSearch indexes for data objects."
+
+    def handle(self, *args, **options):
+        """Command handle."""
+        for data in Data.objects.all():
```

Comment:
Can we avoid hardcoding which models to index? Instead, the list of registered index classes should be used to determine the set of models to index.